### PR TITLE
#160451516  Article report feature

### PR DESCRIPTION
**Pull request by oladimillion**
#### What does this PR do?
This PR add functionality that allows user to report an article

#### Description of Task to be completed?
Write the code that handles article report functionality
Write tests for article report functionality

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
Story type: feature
Story title: Users should be able to report an article
Story ID: #160451516

#### Screenshots (if appropriate)
N/A

#### Questions:```
N/A

### CODE REVIEW

File: server/models/articlecomment.js [hunk 1]
```diff
@@ -16,10 +16,6 @@ export default (sequelize, DataTypes) => {
       targetKey: 'slug',
       onDelete: 'CASCADE',
     });
-    ArticleComment.hasMany(models.Reply, {
```
Comment (oladimillion) [inline server/models/articlecomment.js]: Forgot to remove this part when I was working on article comment reply

**Review by slimsolz — COMMENTED** [review]
LGTM

**Review by onwuvic — COMMENTED** [review]
LGTM

**Review by samdiano — COMMENTED** [review]
LGTM

**Review by s-akinrele — CHANGES_REQUESTED** [review]
As discussed

**Review by s-akinrele — APPROVED** [review]
thanks for the updates!

**Pull request merged**